content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Migrating Container runtime from dockershim to containerd"
+weight: 10
+content_type: task 
+---
+
+- Drain Node
+```
+# replace <node-to-drain> with the name of your node you are draining
+kubectl drain <node-to-drain> --ignore-daemonsets
+```
+- Stop the Docker daemon
+```
+systemctl stop kubelet
+systemctl stop docker
+```
+- Install & start containerd 
+
+
+### For Linux systems
+Use the following commands to install Containerd on your system:
+
+Install and configure prerequisites:
+
+```shell
+cat <<EOF | sudo tee /etc/modules-load.d/containerd.conf
+overlay
+br_netfilter
+EOF
+
+sudo modprobe overlay
+sudo modprobe br_netfilter
+
+# Setup required sysctl params, these persist across reboots.
+cat <<EOF | sudo tee /etc/sysctl.d/99-kubernetes-cri.conf
+net.bridge.bridge-nf-call-iptables  = 1
+net.ipv4.ip_forward                 = 1
+net.bridge.bridge-nf-call-ip6tables = 1
+EOF
+
+# Apply sysctl params without reboot
+sudo sysctl --system
+```
+
+Install containerd:
+
+1. Install the `containerd.io` package from the official Docker repositories. 
+   You can find instructions for installing `containerd` at
+   [Starting Containerd](https://containerd.io/docs/getting-started/#starting-containerd).
+
+2. Configure containerd:
+
+   ```shell
+   sudo mkdir -p /etc/containerd
+   containerd config default | sudo tee /etc/containerd/config.toml
+   ```
+
+3. Restart containerd:
+
+   ```shell
+   sudo systemctl restart containerd
+   ```
```

Comment:
I would explicitly link to the next step the reader should follow, where they start the kubelet etc.

Reply:
+1 - something like "After you install containerd, you must \[configure the kubelet to use containerd as the container runtime\](#use-containerd-as-runtime)."